src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedSdkReference.xaml

```diff
@@ -8,17 +8,18 @@
     xmlns="http://schemas.microsoft.com/build/2009/properties">
     <Rule.DataSource>
         <DataSource Persistence="ResolvedReference" ItemType="SDKReference" HasConfigurationCondition="False"
-                    SourceType="TargetResults" MSBuildTarget="ResolveSDKReferencesDesignTime" SourceOfDefaultValue="AfterContext" />
+                    SourceType="TargetResults" MSBuildTarget="CollectResolvedSDKReferencesDesignTime" />
     </Rule.DataSource>
     <StringProperty Name="AppXLocation" DisplayName="App Package Location" />
-
     <!-- This property should be the same as the one for the ResolvedReference item -->
     <StringProperty Name="OriginalItemSpec" Visible="false" />
-
+    <StringProperty Name="SDKRootFolder" DisplayName="SDK Root" />
```

Comment:
Visible=False for this and other properties below? Note that you have to run [https://github.com/nguerrera/xliff-converter](https://github.com/nguerrera/xliff-converter) to produce localized xaml files as well.